frontend/src/v5/store/projects/projects.sagas.ts

```diff
@@ -17,13 +17,17 @@
 
 import { put, takeLatest } from 'redux-saga/effects';
 import * as API from '@/v5/services/api';
+import { redirectTo404Page } from '@/v5/store/store.helpers';
 import { ProjectsActions, ProjectsTypes, IProject } from './projects.redux';
 
 export function* fetch({ teamspace }) {
 	try {
 		const { data: { projects } } = yield API.fetchProjects(teamspace);
 		yield put(ProjectsActions.fetchSuccess(teamspace, projects as IProject[]));
 	} catch (e) {
+		if (e.status === 404) {
```

Comment:
If I'm reading this right, it'll redirect to 404 page if the api request returned with 404 ?

This shouldn't happen. any 404 from network requests should utilise the error pop up that @m1chalkubiak  is working on.

the 404 page should only be used by for front end routes that are unknown.